### DISCUSSION TITLE
Is it possible to have printed reports in firefly for future dates?

### DISCUSSION THREAD

**Discussion by FSNaval** [firefly-iii/firefly-iii#7263] [General]
Good day all,

i was wondering if it is possible for firefly to print reports containing future transactions that are to come, for example if i wanted to know all the expected transactions for the next six months, how could i do that?

**Comment by JC5** [comment]
https://docs.firefly-iii.org/firefly-iii/more-information/what-its-not/#prediction-forecasting-future-gazing
